nvflare/tool/package_checker/package_checker.py

```diff
@@ -77,35 +84,50 @@ def check(self):
 
             # check if server can run
             if all_passed:
-                self.check_dry_run(timeout=self.dry_run_timeout)
-
+                ret_code = self.check_dry_run(timeout=self.dry_run_timeout)
+            return ret_code
         except Exception as e:
             self.add_report(
                 "Package Error",
                 f"Exception happens in checking: {e}, this package is not in correct format.",
                 "Please download a new package.",
             )
+            return ret_code
```

Comment:
suggest return True/False instead of `ret_code`
Since the typehint in method signature suggest "bool"

Reply:
Or we need to change the type hint to return "int"